tools/cli/admin_task_list_commands.go

```diff
@@ -137,35 +142,74 @@ func AdminListTaskList(c *cli.Context) error {
 		return commoncli.Problem("Operation GetTaskListByDomain failed.", err)
 	}
 
+	if c.String(FlagFormat) == formatJSON {
+		prettyPrintJSONObject(getDeps(c).Output(), response)
+		return nil
+	}
+
 	fmt.Println("Task Lists for domain " + domain + ":")
```

Comment:
Not for this diff, but mixing print statements with the stdout writer is :sadpanda: